jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -119,11 +119,21 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                                 }
 
                                 // Pre-process assistant test scripts
-                                sh '''#!/bin/bash -x
-                                pushd build-config
-                                ./build-config
-                                popd
-                                '''
+                                def repos = env.REPOS_UNDER_TEST.tokenize(',')
```

Comment:
I'm not sure if ```images-service``` need sol log, if needed, please put ```generateSolLog.sh.in``` in to image-service folder